resources/tests.mjs

```diff
@@ -551,5 +551,44 @@ Suites.push({
     ],
 });
 
+Suites.push({
+    name: "Perf-Dashboard",
+    url: "tentative/perf.webkit.org/public/v3/#/charts?since=1678991819934&paneList=((55-1649-53731881-null-(5-2.5-500))-(55-1407-null-null-(5-2.5-500))-(55-1648-null-null-(5-2.5-500))-(55-1974-null-null-(5-2.5-500)))",
+    async prepare(page) {
+        await page.waitForElement("#app-is-ready");
+        page.call('serviceRAF');
```

Comment:
Replacing requestAnimationFrame with setTimeout won't let us force update the charts between `mousemove`, etc... because they'll all become async work if we did that.